### ISSUE TITLE
pdf create + stamp + optimization problem

### ISSUE THREAD

**Issue by adamgreenhall** [pdfcpu/pdfcpu#429]
OS version: MacOS 10.15
latest pdfcpu release: yes

I'm trying to place a PDF on larger size pages and also shift it around by some <x,y>. `pdfcpu stamp` is really useful for this, but I needed a blank PDF to start with, so I've been using `pdfcpu create`. As an example, my workflow is:
```
pdfcpu create two-pages.json out.pdf
pdfcpu stamp add -p 1 -m pdf pkg/testdata/testImage.pdf:1 "pos:c, rot:0"  out.pdf
pdfcpu stamp add -p 2 -m pdf pkg/testdata/testImage.pdf:2 "pos:c, rot:0, offset:150 0"  out.pdf
```
where `two-pages.json` is the simplest possible definition:
```
{
  "paper": "A2",
  "pages": {
    "1": {
      "content": {}
    },
    "2": {
      "content": {}
    }
  },
  "conf": {}
}
```

This was working great (maybe there's a better way to do this? but it works) - until [this commit](https://github.com/pdfcpu/pdfcpu/commit/a0027457ca22754cf8eee01e4734665138af4b14) (#385). After that commit (including latest master), the output pdf has doubled images on both pages. My guess is that the optimization change thinks the two blank pages are identical and somehow that's breaking some logic in stamping about add vs modify.

Here's the pdf output from before the change:
[result-96a0a304946a5e43bbfc440ea9c5681df0b5da7a.pdf](https://github.com/pdfcpu/pdfcpu/files/8019671/result-96a0a304946a5e43bbfc440ea9c5681df0b5da7a.pdf) (correct)
and after:
[result-a0027457ca22754cf8eee01e4734665138af4b14.pdf](https://github.com/pdfcpu/pdfcpu/files/8019675/result-a0027457ca22754cf8eee01e4734665138af4b14.pdf) (incorrect)

Thanks for taking a look! And let me know if I can help with the fix.

**Comment by hhrutter** [comment]
You are right!
Content stream optimization before stamping may be counter productive.
At least empty content streams should not be optimized away.
I need to think about how to handle this.

**Comment by hhrutter** [comment]
So this is what's happening:
The optimize step prior to stamping detects identical (empty) content streams and optimizes them away.
When you stamp something on page 1 you are really also stamping on page 2 because both pages share the same content stream that gets manipulated. You see the unwanted stamp on page 2 only after this page has been stamped and there is a form object defined...

TLDR: Stamping simply does not work on PDFs with optimized content streams.
1) We need to disable content stream optimization when executing the stamp command.
2) The stamp command needs to detect prior optimized content streams and abort.
3) Content stream optimization needs to be a configuration flag that's disabled by default 
(That was the original idea by @maxiride but now I now why we really need this 👍 )

**Comment by hhrutter** [comment]
This should be fixed with the latest commit.

The configuration / config.yml has now a switch for optimizing duplicated content streams:
```
# optimize duplicate content streams across pages
optimizeDuplicateContentStreams: false
```
Please let me know if this works for you.

@maxiride You have to explicitly turn that on!
